src/Api/Utils.purs

```diff
@@ -14,6 +14,7 @@ import Conduit.Data.Username (Username)
 import Conduit.Env (UserEnv)
 import Control.Monad.Reader (class MonadAsk, ask, asks)
 import Data.Argonaut.Core (Json)
+import Data.Bifunctor (rmap)
```

Comment:
This import and two lines (each with comments) are the only changes to code in this file. The rest is noise from my editor removing trailing spaces.